tests/test_fixtures/rule_based_profiler/example_notebooks/DataAssistants_Instantiation_And_Running-OnboardingAssistant-Sql.ipynb

```diff
@@ -0,0 +1,1019 @@
+{
```

Comment:
For easier review.
https://github.com/great-expectations/great_expectations/blob/5fcf5520878a009a09d8b280e604460621ffc17f/tests/test_fixtures/rule_based_profiler/example_notebooks/DataAssistants_Instantiation_And_Running-OnboardingAssistant-Sql.ipynb

Reply:
I did notice there's quite a bit of commented-out code in the notebooks. Is that intentional?

Reply:
Hi @Kilo59 yes, at the end especially there is a section where I describe loading the example `csv` into our `postgres` Docker container. This I've decided to comment out.